index.js

```diff
@@ -21,7 +23,55 @@ mongoose
   })
   .then(() => {
     // Run your code here, after you have insured that the connection was made
+   /* THIS IS ITERATION 2!!!!!!!!!
+    Recipe.create( data[0] )
+      .then((recipe) => {
+        console.log(recipe.title)
+
+      })
+      .catch(()=> {
+        console.log()
+      })
+      */
+
+     Recipe.insertMany(data)
+        .then((recipes) => {
+          //console.log(recipes)
+          for(let recipe in recipes){
```

Comment:
good use of for in loop, but instead of "recipe" a better variable name would be index.